package.json

```diff
@@ -7,8 +7,9 @@
     "changelog": "changelog eslint-plugin-security all > CHANGELOG.md",
     "test": "mocha test/**/*",
     "lint": "eslint .",
```

Comment:
Please also add this to lint-staged so it runs on precommit

Reply:
Done